src/main/java/frc/robot/subsystems/loader/Loader.java

```diff
@@ -39,4 +41,8 @@ public void setVoltage(double voltage) {
     public void move(double power) {
         talonSRX.set(power);
     }
+
+  public boolean getIsInStall() {
+    return talonSRX.getStatorCurrent() > robotConstants.loaderConstants.kThresholdLimit;
```

Comment:
kThresholdLimit is problematic to use... make another constant or calculate the stall by given voltage.
**V = I × R**